src/main/resources/application.yaml

```diff
@@ -77,6 +77,14 @@ features:
     interpreter: ${INTERPRETER_DATA_MIGRATION_ENABLED:false}
     rollback: ${DATA_MIGRATION_ROLLBACK:false}
     encoded-data-string: ${ENCODED_MIGRATION_DATA_STRING:W3sicmVmZXJlbmNlIjoiMTcwMzAyMTkyNDYwMDQxOCIsImludGVycHJldGVyIjoiIFllcyIsInN0YXRlIjoiIHJlYWR5VG9MaXN0ICAgICAgIiwiZXhpc3RpbmdfbGFuZ3VhZ2VfdmFsdWUiOiIgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAiLCJtYXBwZWRfbGFuZ3VhZ2VfdmFsdWUiOiJiZW5nYWxpIn1d}
+    startHour: ${INTERPRETER_MIGRATION_START_HOUR:}
```

Comment:
can we also add endHour for case-loader

Reply:
Think this is already defined as sscs.case.loader.endHour in application.yaml